dev_guide/builds.adoc

```diff
@@ -907,7 +907,9 @@ The source path can be any absolute path within the image specified.  The destin
 must be a relative directory path.  At build time, the image will be loaded
 and the indicated files/directories will be copied into the context directory
 of the build process.  This is the same directory into which the source repository
-content (if any) is cloned.
+content (if any) is cloned.  If the source path ends in `*/.*` then the content
```

Comment:
@bparees 
I don't understand if that is a literal string (asterisk, slash, dot, asterisk), or a glob pattern (any or no character, slash, dot, any or no character).